upi/aws/cloudformation/05_cluster_master_nodes.yaml

```diff
@@ -188,6 +188,8 @@ Resources:
       Tags:
       - Key: !Join ["", ["kubernetes.io/cluster/", !Ref InfrastructureName]]
         Value: "shared"
+      - Key: "Name"
+        Value: !Join ["", [!Ref InfrastructureName, "-master-1"]]
```

Comment:
We usually have them indexed from 0

Reply:
This is only a tagName, that can be used as a reference, but the important there is that by default the Cloudformation Template have not, the proper tagName, and the cluster is deployed without any names into the ec2-instance (its hard to identify which is the proper instance/node).

Reply:
@abhinavdahiya sure! I changed the code, and now are indexed from 0.

Can you please review the changes?